webpack.config.js

```diff
@@ -0,0 +1,64 @@
+import path from 'path';
+import webpack from 'webpack';
+import UglifyJSPlugin from 'uglifyjs-webpack-plugin';
+
+import {settings} from './gulp/config';
+
+const dev = {
+	entry: {
+		polyfills: './src/app/polyfills.js',
+		vendor: './src/app/vendor.js',
```

Comment:
Might be nicer getting the vendor entrypoints from the package.json and pass it as an array.

Reply:
Ah. I didn't know that works 👌